homeassistant/components/template/vacuum.py

```diff
@@ -194,6 +194,8 @@ def __init__(
         if self._battery_level_template:
             self._attr_supported_features |= VacuumEntityFeature.BATTERY
 
+        self._attr_supported_features |= VacuumEntityFeature.STATUS
```

Comment:
You're conflating status with state. Your screenshot shows the state, not the status.